pkg/cmd/server/admin/overwrite_bootstrappolicy.go

```diff
@@ -112,6 +154,10 @@ func (o OverwriteBootstrapPolicyOptions) OverwriteBootstrapPolicy() error {
 		return err
 	}
 
+	if err = o.GateCommand(); err != nil {
```

Comment:
Since we dropped the test that verifies this now, can we have `hack/test-cmd.sh` making sure this actually fails, at least?